src/LockstakeEngine.sol

```diff
@@ -106,18 +107,18 @@ contract LockstakeEngine is Multicall {
     event AddFarm(address farm);
     event DelFarm(address farm);
     event Open(address indexed owner, uint256 indexed index, address urn);
-    event Hope(address indexed urn, address indexed usr);
-    event Nope(address indexed urn, address indexed usr);
-    event SelectVoteDelegate(address indexed urn, address indexed voteDelegate);
-    event SelectFarm(address indexed urn, address farm, uint16 ref);
-    event Lock(address indexed urn, uint256 wad, uint16 ref);
-    event LockNgt(address indexed urn, uint256 ngtWad, uint16 ref);
-    event Free(address indexed urn, address indexed to, uint256 wad, uint256 freed);
-    event FreeNgt(address indexed urn, address indexed to, uint256 ngtWad, uint256 ngtFreed);
-    event FreeNoFee(address indexed urn, address indexed to, uint256 wad);
-    event Draw(address indexed urn, address indexed to, uint256 wad);
-    event Wipe(address indexed urn, uint256 wad);
-    event GetReward(address indexed urn, address indexed farm, address indexed to, uint256 amt);
+    event Hope(address indexed owner, uint256 indexed index, address indexed usr);
+    event Nope(address indexed owner, uint256 indexed index, address indexed usr);
+    event SelectVoteDelegate(address indexed owner, uint256 indexed index, address indexed voteDelegate);
+    event SelectFarm(address indexed owner, uint256 indexed index, address indexed farm, uint16 ref);
+    event Lock(address indexed owner, uint256 indexed index, uint256 wad, uint16 ref);
+    event LockNgt(address indexed owner, uint256 indexed index, uint256 ngtWad, uint16 ref);
+    event Free(address indexed owner, uint256 indexed index, address to, uint256 wad, uint256 freed);
+    event FreeNgt(address indexed owner, uint256 indexed index, address to, uint256 ngtWad, uint256 ngtFreed);
+    event FreeNoFee(address indexed owner, uint256 indexed index, address to, uint256 wad);
+    event Draw(address indexed owner, uint256 indexed index, address to, uint256 wad);
+    event Wipe(address indexed owner, uint256 indexed index, uint256 wad);
+    event GetReward(address indexed owner, uint256 indexed index, address indexed farm, address to, uint256 amt);
```

Comment:
As in this event it is not possible to make `to` indexed due to the amount limit, it was opted to remove the `indexed` from all the `to` values in the prev events to keep consistency. Let me know if preferred to add them back for the rest.